.kokoro/promote.sh

```diff
@@ -38,6 +38,7 @@ create_settings_xml_file $MAVEN_SETTINGS_FILE
   --batch-mode \
   --settings ${MAVEN_SETTINGS_FILE} \
   -DstagingRepositoryId=${STAGING_REPOSITORY_ID} \
-  -Drelease=true
+  -Drelease=true \
+  --activate-profiles skip-unreleased-modules
```

Comment:
Is there a way to test this? Cloud this accidentally mess with other profile activations?

Reply:
> Is there a way to test this

Not really - at least we'd be able to check during the first `stage` step to see if it's included (though it shouldn't be).